liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/CreateServerMojo.java

```diff
@@ -37,6 +41,40 @@
      */
     private String template;
     
+    /**
+     * Packages to install. One of "all", "dependencies" or "project".
+     * 
+     * @parameter property="installAppPackages" default-value="dependencies"
+     * @readonly
+     */
+    private String installAppPackages = "dependencies";
+    
```

Comment:
I am not sure if that is the best thing to do either.  Let's go with it as-is and if we come up with something better, we can do it when we change the appsDirectory default.  I will put a comment in issue #125 to consider it there.